packages/mpl-widget-box.yaml

```
@@ -0,0 +1,5 @@
+repo: leejjoon/mpl_widget_box
+keywords: [widgets]
+section: interactivity
+description: GUI-neutral widgets for Matplotlib w/ legend-like layout and more
```

Looks like we always put a trailing period.
```suggestion
description: GUI-neutral widgets for Matplotlib, with legend-like layout and more.
```